aminator/plugins/provisioner/apt.py

```diff
@@ -134,9 +148,33 @@ def apt_get_update(self):
     def apt_get_clean():
         return monitor_command(['apt-get', 'clean'])
 
+    @staticmethod
+    def apt_get_install(*options):
+        install_result = monitor_command(['apt-get', '-y', 'install'] + options)
+        if not install_result.success:
+            log.debug('failure:{0.command} :{0.std_err}'.format(install_result.result))
+        return install_result
+
     @classmethod
-    def apt_get_install(cls, package):
-        return monitor_command(['apt-get', '-y', 'install', package])
+    def _install(cls, package):
```

Comment:
This is overridden in aptitude since it needs special care given that aptitude doesn't return error codes as you'd expect